src/main/groovy/org/transmartproject/batch/highdim/mrna/data/MrnaDataJobConfiguration.groovy

```diff
@@ -59,10 +59,9 @@ class MrnaDataJobConfiguration extends AbstractStandardHighDimJobConfiguration {
                 partitionByColumn: 'trial_name',
                 partitionByColumnValue: studyId,
                 sequence: Sequences.MRNA_PARTITION_ID,
-                indexes: [
-                        ['assay_id'],
-                        ['probeset_id'],
-                        ['patient_id'],
+                primaryKey: ['assay_id', 'probeset_id'],
+                foreignKeys: [
+                        assay_id: [table: Tables.SUBJ_SAMPLE_MAP, column: 'assay_id', onDelete: 'CASCADE']
```

Comment:
Now that I think about it, the index on `probeset_id` shouldn't be important because it's never used to reduce the size of the result set. And from my tests, under these circumstances, doing a join with ssm after a seq scan or an index for probeset_id makes only a small difference (~20%).